packages/fx-core/templates/plugins/resource/aad/auth/tab/README.md

```diff
@@ -28,11 +28,11 @@ These two HTML files are used for auth redirects.
     `GetUserProfile`: This file implements a function that calls Microsoft Graph API to get user info.
 
 1. Execute the following commands under `tabs/`: `npm install @microsoft/teamsfx-react`
-1. Add the following lines to `tabs/src/components/sample/Welcome.tsx` to import `InitTeamsFx`:
+1. Add the following lines to `tabs/src/components/sample/Welcome.*` to import `InitTeamsFx`:
     ```
     import { InitTeamsFx } from "../../sso/InitTeamsFx";
     ```
-1. Replace the following line: `<AddSSO />` with `<InitTeamsFx />` to replace the `AddSso` component with `InitTeamsFx` component.
+1. Replace the following line: `<AddSSO />` with `<InitTeamsFx />` to replace the `AddSSO` component with `InitTeamsFx` component.
```

Comment:
From triage discussion, Zhen want us to give more pictures to show modifications

Reply:
I tried to add screenshots, but actually I find it will make this document very long. Feel like to better to keep it concise for now.